Core/Core/Inbox/ComposeMessage/View/RecipientFilterView.swift

```diff
@@ -0,0 +1,61 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2024-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+struct RecipientFilterView: View {
+    // MARK: - Properties
+    let recipients: [Recipient]
+    var didSelectRecipient: ((Recipient) -> Void)
+
+    // MARK: - Body
+    var body: some View {
+        ScrollView {
+            VStack {
+                ForEach(recipients, id: \.self) { recipient in
+                    Button {
+                        didSelectRecipient(recipient)
+                    } label: {
+                        RecipientRow(recipient: recipient, showSeparator: recipient != recipients.last)
+                            .frame(height: 50)
+                            .accessibilityLabel(recipient.displayName)
+                            .accessibility(hint: Text("Double tap to select", bundle: .core))
+                    }
+                }
+            }
+            .background(Color.backgroundLightest)
+        }
+        .frame(maxHeight: 160)
```

Comment:
IMHO this is a bit short (3 results max). Also not sure it scales well with increased font sizes.